src/Auth0.ManagementApi/Clients/ConnectionsClient.cs

```diff
@@ -86,29 +86,84 @@ public Task<Connection> GetAsync(string id, string fields = null, bool includeFi
         /// <summary>
         /// Retrieves every connection matching the specified strategy. All connections are retrieved if no strategy is being specified. Accepts a list of fields to include or exclude in the resulting list of connection objects.
         /// </summary>
+        /// <remarks>
+        /// Can only retrieve a maximum of 1000 connections, if you need to retrieve more use the overload with <see cref="CheckpointPagingInformation"/>
+        /// </remarks>
+        /// <param name="request">Specifies criteria to use when querying connections.</param>
+        /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
+        /// <returns>An <see cref="IPagedList{Connection}"/> containing the list of connections.</returns>
+        public Task<IPagedList<Connection>> GetAllAsync(GetConnectionsRequest request, CancellationToken cancellationToken = default)
+        {
+            if (request == null)
+                throw new ArgumentNullException(nameof(request));
+
+            return GetAllAsync(request, new Dictionary<string, string>(), cancellationToken);
+        }
+
+        /// <summary>
+        /// Retrieves every connection matching the specified strategy. All connections are retrieved if no strategy is being specified. Accepts a list of fields to include or exclude in the resulting list of connection objects.
+        /// </summary>
+        /// <remarks>
+        /// Can only retrieve a maximum of 1000 connections, if you need to retrieve more use the overload with <see cref="CheckpointPagingInformation"/>
+        /// </remarks>
         /// <param name="request">Specifies criteria to use when querying connections.</param>
         /// <param name="pagination">Specifies pagination info to use when requesting paged results.</param>
         /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
         /// <returns>An <see cref="IPagedList{Connection}"/> containing the list of connections.</returns>
-        public Task<IPagedList<Connection>> GetAllAsync(GetConnectionsRequest request, PaginationInfo pagination = null, CancellationToken cancellationToken = default)
+        public Task<IPagedList<Connection>> GetAllAsync(GetConnectionsRequest request, PaginationInfo pagination, CancellationToken cancellationToken = default)
         {
             if (request == null)
                 throw new ArgumentNullException(nameof(request));
+            
+            if (pagination == null)
+                throw new ArgumentNullException(nameof(pagination));
```

Comment:
This is a breaking change, anyone that used to do `GetAllAsync(new GetConnectionsRequest(), null)`, will now get an error.

We shouldn't break existing behavior.

Reply:
My last commit should have resolved this issue